esy.json

```diff
@@ -4,18 +4,20 @@
   "license": "MIT",
   "version": "3.5.0",
   "dependencies": {
-    "ocaml": " >= 4.2.0 < 4.9.0",
-    "@opam/ocamlfind": "*",
+    "@opam/dune": "*",
     "@opam/menhir": " >= 20170418.0.0",
-    "@opam/utop": " >= 1.17.0 < 2.3.0",
     "@opam/merlin-extend": " >= 0.4",
-    "@opam/result": "*",
     "@opam/ocaml-migrate-parsetree": "*",
-    "@opam/dune": "*"
+    "@opam/ocamlfind": "*",
+    "@opam/result": "*",
+    "@opam/utop": " >= 1.17.0 < 2.3.0",
+    "@opam/shell": "*",
```

Comment:
https://ocaml.janestreet.com/ocaml-core/latest/doc/shell/Shell/index.html

with shell you can do this:
```
let patch = run_full(~expect=[0, 1], "diff", ["-u", "--", file1, file2]);
```

Reply:
just as an example of what is possible with it, we actually don't do this manually but use rely